specs/lib/loadImageUrisSpec.js

```diff
@@ -0,0 +1,88 @@

```

Comment:
In order to make this `.js` file a lot smaller, can you submit a copy of `Cesium_Logo_Flat.png` that is perhaps 8x8 and then embed that here?